<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transformacje 2D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="rysujFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>580</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zakończ program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>500</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ładuj Obraz</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="translationBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Translation</string>
    </property>
    <widget class="QSlider" name="translationXSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-256</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="translationYSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-256</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scalingBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Scaling</string>
    </property>
    <widget class="QSlider" name="scalingXSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="scalingYSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="scalingCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>98</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Zachowaj początkowe proporcje</string>
     </property>
    </widget>
    <widget class="QSlider" name="scalingAllSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="shearingBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Shearing</string>
    </property>
    <widget class="QSlider" name="shearingXSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="shearingYSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rotationBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotation</string>
    </property>
    <widget class="QSlider" name="rotationSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-360</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
